lightning/src/ln/msgs.rs

```diff
@@ -1698,6 +1698,13 @@ mod fuzzy_internal_msgs {
 			amt_to_forward: u64,
 			outgoing_cltv_value: u32,
 		},
+		#[allow(dead_code)] // TODO (arik): drop in follow-up PR
+		TrampolineEntry {
```

Comment:
IMO the word "Entry" is superfluous. Unless there are multiple trampoline outbound payload variants?

Reply:
Ok, I see. I misunderstood "entry" to mean it's like, an entry into the vec of payloads. Maybe `Entrypoint` or `Initial` would be clearer? Could also add a comment.